app/src/main/java/org/sil/gatherwords/SessionActivity.java

```diff
@@ -18,35 +19,43 @@
 import android.widget.ArrayAdapter;
 import android.widget.EditText;
 import android.widget.Spinner;
+import android.widget.TextView;
 
 import com.google.android.gms.location.FusedLocationProviderClient;
 import com.google.android.gms.location.LocationServices;
 
 import org.sil.gatherwords.room.AppDatabase;
 import org.sil.gatherwords.room.Session;
+import org.sil.gatherwords.room.SessionDao;
 
+import java.lang.ref.WeakReference;
 import java.text.SimpleDateFormat;
 import java.util.Date;
+import java.util.List;
 import java.util.concurrent.ExecutionException;
 
 public class SessionActivity extends AppCompatActivity implements AdapterView.OnItemSelectedListener {
+    EditText dateField, timeField, timeZoneField, labelField, speakerField, eliciterField;
+    SimpleDateFormat dateSDF, timeSDF, timeZoneSDF;
     // Used to track location through multiple methods
     private FusedLocationProviderClient mFusedLocationClient;
     boolean locationEnabled;
     Location location;
     boolean creatingNewSession;
-    AppDatabase db;
+    int id;
     Spinner spinner;
     String worldListToLoad;
 
     public static final String CREATING_SESSION = "creating_session";
+    public static final String ID = "id";
```

Comment:
Perhaps we should decide on a standard for the constants.  In some places we use `... String ARG_XXX = "xxx";` and others (like here) we simply use `... String XXX = "xxx";`.